packages/contracts/scripts/generate-artifacts.ts

```diff
@@ -15,6 +15,15 @@ const main = async () => {
 
   const artifactPaths = glob
     .sync(`${contractArtifactsFolder}/**/*.json`)
+    .filter((match) => {
+      return [
+        'artifacts/contracts/chugsplash',
```

Comment:
These are now the only artifact folders contained in the NPM build. Please reference `package.json` to confirm this.

Reply:
Looking here: https://github.com/ethereum-optimism/optimism/blob/59e41374800dedec872bd4ef4211c057c8badd29/packages/contracts/package.json#L11

Appears to match

Reply:
resolving to get on the merge train